doc/faq.rst

```diff
@@ -0,0 +1,283 @@
+
+
+.. dropdown:: Q: How do I clear all variables before executing my code?
+
+   Check the option :guilabel:`Remove all variables before execution` in the :guilabel:`Configuration per file...` dialog under the :guilabel:`Run` menu.
+
+   .. image:: images/faq/faq-remove-variables.png
+      :alt: Spyder showing cell generation.
+
+
+.. dropdown:: Q: How do I run my code in a dedicated console or an external system terminal?
+
+   Select the appropriate option in the :guilabel:`Configuration per file...` dialog under the :guilabel:`Run` menu.
+
+   .. image:: images/faq/faq-run-options.png
+      :alt: Spyder showing cell generation.
+
+
+.. dropdown:: Q: How do I change the syntax highlighting theme?
+
+   Go to :guilabel:`Preferences` and select the theme you want under :guilabel:`Syntax highlighting theme` in the :guilabel:`Appearance` section.
+
+   .. image:: images/faq/faq-highlighting-theme.png
+      :alt: Spyder showing cell generation.
+
+
+
+===============
+Troubleshooting
+===============
+
+.. dropdown:: Q: I've found a bug or issue with Spyder. What do I do?
+
+   You should first follow the steps in our :doc:`troubleshooting guide<first-steps>`.
+   If you can't solve your problem, open an issue by following the instructions in our :doc:`submit-a-report` section.
+
+
+.. dropdown:: Q: I get an error in the IPython console running my code! Help!
+
+   First, make sure the error you are seeing is not a bug in your code.
+   To confirm this, try running it in any standard Python interpreter.
+   If the error still occurs, the problem is likely with your code and a site like `Stack Overflow`_ might be the best place to start.
+   Otherwise, start at the :doc:`basic-first-aid` section of our troubleshooting guide.
+
+   .. _Stack Overflow: https://stackoverflow.com
+
+
+.. dropdown:: Q: Code completion/help doesn't work; what can I do?
+
+   If nothing is displayed in the calltip, hover hint or :doc:`help` pane, make sure the object you are inspecting has a docstring, and try executing your code in the :doc:`ipythonconsole` to get help and completions there.
+   If this doesn't work, try restarting PyLS by right-clicking the :guilabel:`LSP Python` label item in the statusbar at the bottom of Spyder's main window, and selecting the :guilabel:`Restart Python Language Server` option.
+
+   For more information, go to the Completion/help not working section in the :doc:`common-illnesses` page of our troubleshooting guide.
+
+
+.. dropdown:: Q: I get the message "An error occurred while starting the kernel". How do I fix this?
+
+   First, make sure your version of Spyder-Kernels is compatible with that of Spyder.
+
+    .. table::
+
+       ==============   ==============
+       Spyder           Spyder-Kernels
+       ==============   ==============
+       4.0.0-4.0.1      1.8.1
+       4.1.0-4.1.2      1.9.0
+       4.1.3            1.9.1
+       4.1.4            1.9.2
```

Comment:
```suggestion
       4.1.4            1.9.3
```

Reply:
My suggestion must not have gotten applied